tests/integrate/217_NO_out_pot_nscf/INPUT

```diff
@@ -0,0 +1,27 @@
+INPUT_PARAMETERS
+#Parameters (1.General)
+suffix			autotest
+calculation     nscf
+
+nbands			6
+symmetry		1
+pseudo_dir	../../PP_ORB
+orbital_dir	../../PP_ORB
+out_pot			1
+
+#Parameters (2.Iteration)
+ecutwfc			20
```

Comment:
Could you further reduce ecutwfc from 20 to 5? I would like to reduce the size of the .cube file.

Reply:
ok, done.